crates/pallet-humanode-session/src/lib.rs

```diff
@@ -29,7 +29,6 @@ pub mod pallet {
     #[pallet::config]
     pub trait Config:
         frame_system::Config
-        /* Session pallet is only required for migration to v1. */
```

Comment:
Why remove this?

Reply:
As we need to use `T::ValidatorIdOf` as a part of `frame_session::Config` to be passed to `<pallet_session::Pallet<T>>::disable(..)`.

So, it's not just used for migration.

Reply:
Not removed now.